src/commons/report.py

```diff
@@ -106,6 +107,8 @@ def update_tests_status(input_dict: dict, corio_start_time: datetime, value: dic
         if datetime.now() > (test_start_time + value['min_runtime']):
             pass_time = (test_start_time + value['min_runtime']).strftime('%Y-%m-%d %H:%M:%S')
             input_dict["RESULT_UPDATE"] = f"Passed at {pass_time}"
+            if ast.literal_eval(os.getenv("sequential_run")):
+                input_dict["TOTAL_TEST_EXECUTION"] = value['min_runtime']
```

Comment:
Total_test_execution wont be updated in case if its not sequential run ?

Reply:
It should work in either case. Added minor change in condition.